### PR TITLE
Add image megapixel value in the title

### PR DESCRIPTION
**Pull request by darkred**
This is a suggestion I had made to the original script ([link](https://greasyfork.org/en/forum/discussion/11683/x)). 

For reference:
> 1 Megapixel = 1,000,000 pixels
> How to find the megapixel value: 
> multiply the resolution of the image (width * height) then divide by 1,000,000.

In my PR the MP value is rounded to 2 decimal places at most.

Example comparison:
Before:

After:

I believe that the megapixel value is useful info: it helps easily perceive whether (and how much) the size of an image is larger than another.

I hope you like my suggestion.

P.S. Thank you for continuing this great usercript!

### CODE REVIEW

File: script.user.js [hunk 1]
```diff
@@ -15,7 +15,7 @@
 // @grant       GM_openInTab
 // @grant       GM_registerMenuCommand
 
-// @version     1.0.14
+// @version     1.0.15
```
Comment (tophf) [inline script.user.js]: for noncritical stuff I increase the version separately so no need to change it here

File: script.user.js [hunk 2]
```diff
@@ -565,7 +565,8 @@ class App {
   static updateTitle() {
     if (!ai.bar)
       return;
-    const zoom = `${Math.round(ai.scale * 100)}% - ${ai.nwidth} x ${ai.nheight} px`;
+    const megapixels = Math.round(100 * (ai.nwidth * ai.nheight / 1000000)) / 100;
```
Comment (tophf) [inline script.user.js]: I usually write these long numbers as thousands `1000e3` or millions `1e6` to make the order of magnitude more obvious. What do you think?

File: script.user.js (hunk 1 above)
Comment (darkred) [reply script.user.js]: Ok 👍

File: script.user.js (hunk 2 above)
Comment (darkred) [reply script.user.js]: I totally agree. I just made the change.

**Pull request merged**